app/src/main/java/com/distillery/interview/ui/fragments/WeatherObjectFragment.kt

```diff
@@ -0,0 +1,31 @@
+package com.distillery.interview.ui.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import com.distillery.interview.R
+
+const val ARG_OBJECT = "object"
```

Comment:
This can live within a companion object.

Reply:
Agree! But I will remove this variable because is not useful.